### ISSUE TITLE
Some problems about Fine-tuning Biobert

### ISSUE THREAD

**Issue by Peter857** [dmis-lab/biobert-pytorch#1]
Hello, I have been looking at the content of Biobert recently, and I want to fine-tune Biobert, but during the process, how to fill in model_name_or_path. Are there any questions about the following? :
model_name_or_path: str = field(
        metadata={"dmis-lab/biobert-base-cased-v1.1": "Path to pretrained model or model identifier from huggingface.co/models"}
    )
Hope to get your answer, thank you

**Comment by abhibisht89** [comment]
@Peter857  you can give is as dmis-lab/biobert-base-cased-v1.1 \, it will work

python run_ner.py \
    --data_dir ${DATA_DIR}/${ENTITY} \
    --labels ${DATA_DIR}/${ENTITY}/labels.txt \
    --model_name_or_path dmis-lab/biobert-base-cased-v1.1 \
    --output_dir output/${ENTITY} \
    --max_seq_length 128 \
    --num_train_epochs 30 \
    --per_device_train_batch_size 32 \
    --save_steps 1000 \
    --seed 1 \
    --do_train \
    --do_eval \
    --do_predict \
    --overwrite_output_dir

**Comment by abhibisht89** [comment]
@Peter857 is this resolve your issue?
